go.mod

```diff
@@ -34,6 +34,8 @@ require (
 	github.com/mattn/go-sqlite3 v1.14.18
 	github.com/moby/moby v24.0.7+incompatible
 	github.com/moby/term v0.5.0
+	github.com/mrjones/oauth v0.0.0-20190623134757-126b35219450
```

Comment:
That's a pretty old one suggesting to use https://pkg.go.dev/golang.org/x/oauth2#section-readme instead

Reply:
It should be in use for gitea as well if I remember correctly